container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -3,10 +3,10 @@ releases:
   version: "1.0.4"
   url: "https://bosh.io/d/github.com/cloudfoundry/bpm-release?v=1.0.4"
   sha1: "c2cceb2d1e271a2f7c5e7c563a7b26f919ebc17a"
-- name: capi
-  version: "1.79.0"
-  url: "https://bosh.io/d/github.com/cloudfoundry/capi-release?v=1.79.0"
-  sha1: "85d0765ad9815962e46fc99712294c21514e6f7f"
+- name: "capi"
+  version: "1.83.0"
```

Comment:
We should not bump CAPI outside of bumping cf-deployment. CAPI-1.83.0 was not included until cf-deployment-9.4, but the `develop` branch is still based on cf-deployment-7.11. Unmerged PR #2610 is only bumping to cf-deployment-8.1.

So this is fine for testing, but should not be merged to `develop`. There are not even green CATS run on Jenkins using the default Diego scheduler.

Reply:
> We should not bump CAPI outside of bumping cf-deployment. 

Thanks, Didn't know that, good to know, will keep it in mind :) 

> So this is fine for testing, but should not be merged to `develop`. There are not even green CATS run on Jenkins using the default Diego scheduler.

I see the point, I agree.

Should we have an "official" eirini branch until cf-deployment-9.4 is on develop? I say this just because other Eirini features will depend on the changes on this branch

Reply:
> because other Eirini features will depend on the changes on this branch

Is the bump to CAPI-1.83 required for the Eirini changes? So far the plan [CAP-460] for the CAP-1.5 release only includes a bump to cf-deployment-8.1.

I think we do want to include the latest Eirini changes, so if that requires bumping to 9.4 (or 9.latest), then we should do that as soon as possible.

Reply:
Unfortunately we need that - capi 1.81 is ok with the eirini we have currently in develop, but if we want to consume the latest changes for the staging mechanism we have to switch to >=capi 1.83. 

1.82 is the minimum afaict as contains fixes to make available environment variables set from cf push into staging https://github.com/cloudfoundry/capi-release/commit/68a02c3e983348086d9dcde82420a91ea4603e6f (and in this way, Eirini starts to pick up the buildpack dependencies related to the stack which is being set as default. This is why we need now to specify the stack directly and before we didn't had to). 

I will try to see if pinning to 1.82 is possible, as far as I can remember it didn't worked.

Reply:
Update: I can confirm that 1.82 is enough (and necessary) https://github.com/SUSE/scf/pull/2616 , but won't make much difference from 1.83 ( https://github.com/cloudfoundry/cf-deployment/releases/tag/v9.3.0 )

Reply:
I'm attempting an exploratory bump to 9.4 here: https://github.com/SUSE/scf/pull/2618 It is just WIP and to check out if this is the same failures as the 8.1 bump PR (as now both seems to timeouts on cats ssh). I could deploy with Eirini and smoke passed there

Reply:
We now have a bump to 9.5 in https://github.com/SUSE/scf/pull/2610, which has passed all tests on Jenkins.

I would have merged it, but it includes a submodule update for UAA that points to a branch. That needs to be a PR that is merged first.

I expect all this to be worked out on Monday, but maybe you can get a head start by rebasing on top of that PR?

Reply:
Awesome, thanks! Will try and keep you updated

Reply:
I have rebased the branch on top of cf deployment bump (9.5) here: https://github.com/SUSE/scf/pull/2623 . It works locally with Eirini (smoke tests) and cflinuxfs3, and Jenkins passed!

The rebase wasn't enough, I had to revert also https://github.com/SUSE/scf/pull/2623/commits/e2985c967317abac41f4d91130f30cdaae1c1a74 to have Eirini working again: https://github.com/SUSE/scf/pull/2623/commits/924733205c6a699a39b46baf6d3ff4a5524cb245

Otherwise any app push was stuck with:
```
Unexpected Response
Response code: 503
CC code:       0
CC error code: 
Request ID:    3088c643-d781-4177-4b34-79edf9d80e54::c012d48d-b794-4c42-8ee2-a05718a7081b
Description:   {
  "description": "Stager is unavailable: getaddrinfo: Name or service not known (diego-api-bbs.scf.svc.cluster.local:8889)",
  "error_code": "CF-StagerUnavailable",
  "code": 170010
}
```

The cloud controller config file wasn't configured to use eirini.

Reply:
I've not merged today because the cf-deployment-9.5 bump was missing a corresponding update to CATS, which unfortunately breaks. Adnreas is working on a fix (by adding the reverse-log-proxy to SCF, which will eventually replace the firehose).

Once the bump is merged, I will rebase your changes on top of it, and merge them as well.

I've already started some additional work based on top of your current branch, so it is soon going to feel a little bit like Jenga. 😄

Reply:
> I've not merged today because the cf-deployment-9.5 bump was missing a corresponding update to CATS, which unfortunately breaks. Andreas is working on a fix (by adding the reverse-log-proxy to SCF, which will eventually replace the firehose).
> 
> Once the bump is merged, I will rebase your changes on top of it, and merge them as well.
> 

Awesome! thank you @jandubois and @andreas-kupries for your help on this :+1: 

> I've already started some additional work based on top of your current branch, so it is soon going to feel a little bit like Jenga. smile

![Jenga](https://media.giphy.com/media/EuuTtFfRsS6xW/giphy-downsized.gif)

Serious Jenga! :smile: